processor/src/decoder/mod.rs

```diff
@@ -241,6 +241,35 @@ where
         self.execute_op(Operation::Noop)
     }
 
+    // DYN BLOCK
+    // --------------------------------------------------------------------------------------------
+
+    /// Starts decoding of a DYN block.
+    pub(super) fn start_dyn_block(
+        &mut self,
+        block: &Dyn,
+        dyn_hash: Word,
+    ) -> Result<(), ExecutionError> {
+        let addr =
+            self.chiplets
+                .hash_control_block([ZERO; 4], [ZERO; 4], Dyn::DOMAIN, block.hash());
```

Comment:
minor nit: I'd replace `[ZERO; 4]` with `EMPTY_WORD` to be consistent with the changes we made recently in #1059.